packages/web/src/flash/index.ts

```diff
@@ -1,4 +1,5 @@
-import Flash from './Flash'
-import { FlashProvider, useFlash } from './FlashContext'
+export { default as Flash } from './Flash'
+export { useFlash } from './FlashContext'
 
-export { FlashProvider, useFlash, Flash }
+export { default as toast } from 'react-hot-toast'
+export * from 'react-hot-toast'
```

Comment:
Can we split these out into their own file, so that this file can be deleted in v1.0, where we fully remove support for `useFlash` etc? Like maybe create packages/web/src/toast/index.ts and put it there. And then require that file as well in index.js, if possible

Reply:
I like that, then we don't have to parse this when we remove it.